.github/workflows/test-linux-cpu.yml

```diff
@@ -0,0 +1,57 @@
+name: Unit-tests on Linux CPU
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+env:
+  CHANNEL: "nightly"
+jobs:
+  tests:
+    name: "Unit-tests on Linux CPU"
+    runs-on: [self-hosted, linux.8xlarge]
```

Comment:
I think there is no linux.8xlarge according to https://github.com/pytorch/test-infra/blob/main/.github/scale-config.yml, next tier is linux.12xlarge